### PR TITLE
Remove null coalescing operator from layout.

### PR DESCRIPTION
**Pull request by tellthemachines**
Trac ticket: https://core.trac.wordpress.org/ticket/59443

Follow-up to #5252: replaces uses of null coalescing operator in that PR due to it [not being officially supported](https://core.trac.wordpress.org/ticket/58874) in WP yet.

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by dmsnell — APPROVED** [review]

File: src/wp-includes/block-supports/layout.php [hunk 1]
```diff
@@ -549,7 +549,7 @@ function wp_get_layout_style( $selector, $layout, $has_block_gap_support = false
 function wp_render_layout_support_flag( $block_content, $block ) {
 	$block_type            = WP_Block_Type_Registry::get_instance()->get_registered( $block['blockName'] );
 	$block_supports_layout = block_has_support( $block_type, 'layout', false ) || block_has_support( $block_type, '__experimentalLayout', false );
-	$layout_from_parent    = $block['attrs']['style']['layout']['selfStretch'] ?? null;
+	$layout_from_parent    = isset($block['attrs']['style']['layout']['selfStretch']) ? $block['attrs']['style']['layout']['selfStretch'] : null;
```
Comment (mukeshpanchal27) [inline src/wp-includes/block-supports/layout.php]:
```suggestion
	$layout_from_parent    = isset( $block['attrs']['style']['layout']['selfStretch'] ) ? $block['attrs']['style']['layout']['selfStretch'] : null;
```

File: src/wp-includes/block-supports/layout.php [hunk 2]
```diff
@@ -770,7 +770,7 @@ function wp_render_layout_support_flag( $block_content, $block ) {
 	 * @var string|null
 	 */
 	$inner_block_wrapper_classes = null;
-	$first_chunk                 = $block['innerContent'][0] ?? null;
+	$first_chunk                 = isset($block['innerContent'][0]) ? $block['innerContent'][0] : null;
```
Comment (mukeshpanchal27) [inline src/wp-includes/block-supports/layout.php]:
```suggestion
	$first_chunk                 = isset( $block['innerContent'][0] ) ? $block['innerContent'][0] : null;
```

**Review by mukeshpanchal27 — APPROVED** [review]
Some nit-pick feedback.

**Comment by tellthemachines** [comment]
Committed in [r56700](https://core.trac.wordpress.org/changeset/56700).

**Pull request closed without merging**